src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -933,14 +933,18 @@ machine(MachineType:TCC, "TCC Cache")
   }
 
   action(pa_performAtomic, "pa", desc="Perform atomic") {
```

Comment:
Which transition were you seeing this being called after the queue was popped? I don't see on in my code. If it's for some research code that's fine. Generally the pop is the last action though.